src/keycloak/admin/groups.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+from keycloak.admin import KeycloakAdminBase
+
+__all__ = ('Groups',)
+
+
+class Groups(KeycloakAdminBase):
+    _paths = {
+        'groups': '/auth/admin/realms/{realm}/groups',
```

Comment:
Just a very minor issue but in all other classes this key is called "collection" since it's the collection entry point of the groups.

```suggestion
        'collection': '/auth/admin/realms/{realm}/groups',
```